README.md

```diff
@@ -11,6 +11,34 @@
 - A NFT contract instance must validate provided metadata against the specified metadata schema for that contract.
 - Standardized session code to interact with an NFT contract instance must be usable as is, so that a given DApp developer doesn't have to write any Wasm producing logic for normal usage of NFT contract instances produced by this contract.
 
+## New in Version 1.1 {#new-version}
+
+The release of version 1.1 for the CEP-78 Enhanced NFT Standard includes the following:
+
+* [Gas stabilization through the use of of page dictionaries](#data-storage-and-gas-stabilization).
+
+* `OwnerReverseLookupMode` Modality - A modality that allows lookup of which NFTs are owned by a given account or contract. [More information is available here](#ownerreverselookupmode). If this modality is set to `NoLookup`, tokens will retain ownership data, but this data will not be natively indexed.
+
+* Owners must be registered prior to minting **or receiving a transferred token** as an account must pay upfront for page allocation.
+
+* A single instance of CEP-78 is limited to 1,000,000 tokens maximum.
+
+    * Multiple instances of CEP-78 installed from a single account/contract must differentiate their name.
+
+    * If you require multiple contracts for a CEP-78 implementation, we suggest differentiating the contracts by following this syntax: `<collection name>_series_1` and incrementing the number with each instance.
+    
+    * **If an account attempts to install a second CEP-78 contract with the same name, it will overwrite the access rights and render the first instance unusable.**
+
+* New entry points related to the gas stabilization efforts.
+
+  * `register_owner` - Registers an owner with a given instance of CEP-78.
+
+  * `updated_receipts` - Allows an owner to [update their receipts](#updated-receipts) to the new CEP-78 1.1 receipt standard.
+
+  * `migrate` - An entrypoint used to update a CEP-78 version 1.0 contract to the new version. Sending a deploy with *contract.wasm* will initiate the migration process, adding the 1.1 contract version to the contract package and will invoke the `migrate` entry point in order to perform the necessary data migration.
```

Comment:
It would be nice to add usage details for the new entry points, especially for the “migrate” one.

Reply:
Unless we rely on the JS client for examples, that works too.

Reply:
`migrate` is only used through a deploy with a *contract.wasm*, if there's a v1.0 contract it will migrate rather than install.

Reply:
https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/108 addresses what I was looking for as well.